### PR TITLE
Add palantir==1.3.3

### PR DESCRIPTION
**Pull request by katosh**
This PR adds [Palantir v1.3.3](https://github.com/dpeerlab/Palantir) to the bioconda repository.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by katosh** [comment]
@BiocondaBot please add label

File: recipes/palantir/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - scikit-learn
+    - scipy >=1.3
+
+test:
+  imports:
+    - palantir
+
+about:
+  home: "https://github.com/dpeerlab/palantir"
+  license: GPL-2.0
```
Comment (martin-g) [inline recipes/palantir/meta.yaml:57]:
```suggestion
  license: GPL-2.0-only
```

File: recipes/palantir/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  license_family: GPL
+  license_file: 
+  summary: "Palantir for modeling continuous cell state and cell fate choices in single cell data"
+  doc_url: "https://palantir.readthedocs.io"
+  dev_url:
```
Comment (martin-g) [inline recipes/palantir/meta.yaml:62]:
```suggestion
  dev_url: "https://github.com/dpeerlab/palantir"
```

File: recipes/palantir/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+test:
+  imports:
+    - palantir
+
+about:
+  home: "https://github.com/dpeerlab/palantir"
+  license: GPL-2.0
+  license_family: GPL
+  license_file:
```
Comment (martin-g) [inline recipes/palantir/meta.yaml:59]:
```suggestion
  license_file: LICENSE
```

**Comment by katosh** [comment]
Thank you for the review and your suggestion! I adopted all of them.

**Comment by katosh** [comment]
I changed the license back to GPL-2.0-only to avoid any concerns when merging. Please let me know if there are any blockers.

File: recipes/palantir/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 915bb802e01f282d309fab02062caa4e790535279c14f6c15823cdaa01ba7a56
+
+build:
+  number: 0
```
Comment (martin-g) [inline recipes/palantir/meta.yaml:13]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/palantir/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  run_exports:
+        - {{ pin_subpackage('palantir', max_pin="x") }}
+
+requirements:
+  host:
+    - anndata >=0.8.0
+    - fcsparser >=0.1.2
+    - joblib
+    - leidenalg >=0.9.1
+    - matplotlib >=2.2.2
```
Comment (martin-g) [inline recipes/palantir/meta.yaml:24]:
```suggestion
    - matplotlib-base >=2.2.2
```

File: recipes/palantir/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    - python
+    - scanpy >=1.6.0
+    - scikit-learn
+    - scipy >=1.3
+  run:
+    - anndata >=0.8.0
+    - fcsparser >=0.1.2
+    - joblib
+    - leidenalg >=0.9.1
+    - matplotlib >=2.2.2
```
Comment (martin-g) [inline recipes/palantir/meta.yaml:40]:
```suggestion
    - matplotlib-base >=2.2.2
```

**Review by martin-g — APPROVED** [review]

**Pull request merged**